### ISSUE TITLE
🐞 `uninitialized constant Appsignal::Hooks::SidekiqPlugin::Set`

### ISSUE THREAD

**Issue by bradrobertson** [appsignal/appsignal-ruby#372]
This is running on alpine-linux 2.4. It looks like `Set` isn't being required before being used?

I also get this on OSX ruby 2.3. `Set` is stdlib so I think it needs to be required first.

```bash
$ appsignal notify_of_deploy --user=$GITLAB_USER_EMAIL --environment=$DCOS_CLUSTER --revision=$CI_BUILD_REF
/usr/local/bundle/gems/appsignal-2.4.1/lib/appsignal/hooks/sidekiq.rb:25:in `<class:SidekiqPlugin>': uninitialized constant Appsignal::Hooks::SidekiqPlugin::Set (NameError)
Did you mean?  Net
	from /usr/local/bundle/gems/appsignal-2.4.1/lib/appsignal/hooks/sidekiq.rb:22:in `<class:Hooks>'
	from /usr/local/bundle/gems/appsignal-2.4.1/lib/appsignal/hooks/sidekiq.rb:4:in `<module:Appsignal>'
	from /usr/local/bundle/gems/appsignal-2.4.1/lib/appsignal/hooks/sidekiq.rb:3:in `<top (required)>'
	from /usr/local/lib/ruby/site_ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /usr/local/lib/ruby/site_ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /usr/local/bundle/gems/appsignal-2.4.1/lib/appsignal/hooks.rb:100:in `<top (required)>'
	from /usr/local/lib/ruby/site_ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /usr/local/lib/ruby/site_ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /usr/local/bundle/gems/appsignal-2.4.1/lib/appsignal.rb:730:in `<top (required)>'
	from /usr/local/lib/ruby/site_ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /usr/local/lib/ruby/site_ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /usr/local/bundle/gems/appsignal-2.4.1/lib/appsignal/cli.rb:4:in `<top (required)>'
	from /usr/local/lib/ruby/site_ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /usr/local/lib/ruby/site_ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /usr/local/bundle/gems/appsignal-2.4.1/bin/appsignal:4:in `<top (required)>'
	from /usr/local/bundle/bin/appsignal:23:in `load'
	from /usr/local/bundle/bin/appsignal:23:in `<main>'
ERROR: Job failed: exit code 1
```

**Comment by tombruijn** [comment]
Hi @bradrobertson, Thanks for reporting! Worst thing is that it doesn’t even need to be a set either. Previously this was a set defined in method and it worked because something loaded “set” before it was called.

 I recently moved it to the constant in PR #362  and now it’s broken 😓

Will fix tomorrow!

**Comment by tombruijn** [comment]
@bradrobertson the fix for this is released in Ruby gem 2.4.2. Let us know if you run into any other problems!

**Comment by bradrobertson** [comment]
Wow thanks so much for the quick fix! You just unblocked my task :)
